baekjoon/fibonacci.js

```diff
@@ -0,0 +1,12 @@
+let input = require('fs').readFileSync('/dev/stdin').toString();
+const value = Number(input);
+
+function fibonacci(value) {
+    if (!value) return 0;
```

Comment:
```suggestion
    if (value === 0) return 0;
```
제 생각으로는 !value도 좋지만 이 상황에서는 value가 0일때 저 0을 반환하는 거니까 value === 0이 조금 더 직관적이지 않나 싶어요